src/Views/Game.hs

```diff
@@ -39,6 +39,8 @@ createGameDOM api gapi = do
     [
       prop "id" =: "startGameButton"
     ]
+  setClass createStartGameBtn "btn" True
```

Comment:
I think we used prop in the early stages of the project and it just hasn't been changed since then :P 

Never even thought about classes as attributes, I guess they can be set that way :dancer: